src/chapters/cloudfront/cloudfront-fundamentals.js

```diff
@@ -0,0 +1,124 @@
+export const cloudfrontFundamentalsChapter = {
+  id: "cloudfront-fundamentals",
+  title: "CloudFront Fundamentals",
+  sectionId: "cloudfront",
+  previousChapterId: null,
+  content: `## Welcome Back to the Cloud
+
+In our last module, we took our first step into cloud development by deploying our Rock of Ages application to Amazon S3. You created a bucket, configured it for website hosting, and uploaded your React application files. By the end of that module, you had a working website accessible via your S3 bucket's website endpoint.
+
+Today, we're going to enhance your cloud deployment with Amazon CloudFront. This addition will significantly improve your application's performance, security, and global accessibility.
+
+## What We'll Accomplish Today
+
+By the end of this module, your Rock of Ages application will be:
+- Served through a global content delivery network
+- Available with HTTPS security
```

Comment:
Still working on the HTTPS part. I got it working but its going to add some complexity and "hacky" components when we get to Docker and Ec2 so we may not want to include it. We could have something about HTTPS being an available option